src/pages/STG_PanelRDCustomInstallment.page

```diff
@@ -1,64 +1,67 @@
 <apex:page controller="STG_PanelRDCustomInstallment_CTRL" docType="html-5.0"  >
-    <c:STG_PageHeader sectionLabel="{!$Label.stgNavRecurringDonations}" pageLabel="{!$Label.stgNavRDInstallmentPeriods}" />
-
-    <apex:form id="form" styleClass="stgForm form-horizontal">  
-        <apex:pageMessages id="pageMessages"/>
-        <apex:pageBlock mode="maindetail">
-            <apex:outputPanel layout="block" rendered="{!isReadOnlyMode}" styleClass="button-block">
-                <apex:commandButton value="{!$Label.stgBtnNewRDPeriod}" action="{!newCustomInstallment}" status="statusLoad" rendered="{!isReadOnlyMode}" immediate="true" rerender="form" styleClass="btn-sm btn-primary" />
-            </apex:outputPanel>
-
-            <apex:outputPanel layout="block" rendered="{!isEditMode}" styleClass="inline-form-block">
-            <section>
-                <h3 class="section-header" >{!$Label.stgLabelRDNewPeriod}</h3>
-                <div class="form-group">
-                    <apex:outputLabel value="{!$ObjectType.npe03__Custom_Installment_Settings__c.Fields.Name.Label}" for="tbxN" styleClass="col-md-4 control-label" />
-                    <div class="col-md-8 form-control-column">
-                        <apex:inputField value="{!ciNew.Name}" rendered="{!isEditMode}" required="false" id="tbxN" styleClass="form-control" />
-                    </div>
-                    <div class="col-md-offset-4 col-md-8 help-block">
-                        <apex:outputText value="{!$Label.stgHelpRDPeriodName}" />
+    <apex:form id="form" styleClass="slds-m-around--x-large">
+        <c:STG_PageHeader sectionLabel="{!$Label.stgNavRecurringDonations}" pageLabel="{!$Label.stgNavRDInstallmentPeriods}" />
+        <c:UTIL_PageMessages />
```

Comment:
The messages displayed by this component are using too large a font, that seems way out of proportion to our settings page.  Look at this example and see what I mean.
![image](https://cloud.githubusercontent.com/assets/908119/13762085/dc27270a-e9f9-11e5-9ebb-5079fe5ab838.png)